scripts/morty/morty_pde_ode_compare.py

```diff
@@ -0,0 +1,413 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from time import time
+import os
+
+
+class FormatAssist:
+    """
+    Mixin class
+
+    """
+
+    def _format_spatial(self, term1, term2, vector_form=False):
+        """
+        Distribute term 1 in < z1, and term 2 in > z1.
+        Returns a list of terms corresponding to each z.
+        If term1 and term2 are vectors, use `vector_form=True`
+
+        Parameters
+        ----------
+        term1 : float
+            Term in the in-core region
+        term2 : float
+            Term in the ex-core region
+        vector_form : bool
+            Terms are given as spatial vectors rather than floats
+
+        Returns
+        -------
+        return_list : 1D numpy array
+            Spatial distribution of values at each point
+
+        """
+        return_list = []
+        for zi, z in enumerate(self.zs):
+            if z < self.z1:
+                if not vector_form:
+                    return_list.append(term1)
+                else:
+                    return_list.append(term1[zi])
+            elif z > self.z1:
+                if not vector_form:
+                    return_list.append(term2)
+                else:
+                    return_list.append(term2[zi])
+            else:
+                if not vector_form:
+                    return_list.append(0.5 * (term1 + term2))
+                else:
+                    return_list.append(0.5 * (term1[zi] + term2[zi]))
+        return np.asarray(return_list)
+
+
+class DiffEqComp(FormatAssist):
+    def __init__(self, mu, S, isotope, nu1, nu2, z1, z2,
+                 nodes, tf, dt, initial_guess, lmbda):
+        """
+        This class generates results for comparing differential equations.
+        Particularly, this class compares ODE depletion in time to
+        PDE depletion in 1D space and time.
+
+        Parameters
+        ----------
+        mu : dict
+            key : string
+                Name of isotope
+            value : List of float
+                Value of loss in-core and value ex-core
+        S : dict
+            key : string
+                Name of isotope
+            value : List of float
+                Value of source in-core and value ex-core
+        isotope : string
+            Name of isotope to model
+        nu1 : float
+            Linear flow rate in-core
+        nu2 : float
+            Linear flow rate ex-core
+        z1 : float
+            Linear height of in-core region
+        z2 : float
+            Linear height of ex-core region
```

Comment:
```suggestion
        z1 : float
            Linear height of in-core region
        z2 : float
            Linear height of ex-core region
```

These should have more descriptive names. `z1` seems like it refers to a value at a specific index. Additionally, it doesn't match the verbosity of other variable names (e.g., isotope, nodes, initial_guess).